<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>695</width>
    <height>577</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowOpacity">
   <double>0.980000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
	background-color: rgba(65, 65, 65, 250);
}
QFrame{
	
	background-color: rgb(78, 78, 78);
}
QPushButton{
	background-color: rgba(103,103,103,253);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>671</width>
      <height>251</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica Neue</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="genParticleDistButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>200</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Helvetica Neue</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Generate Particle Distribution</string>
     </property>
    </widget>
    <widget class="QPushButton" name="genSpecDistBut">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>200</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Helvetica Neue</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Generate Specific Distribution</string>
     </property>
    </widget>
    <widget class="QPushButton" name="quickPlotBut">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>200</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Helvetica Neue</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Quick SPD</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>140</y>
       <width>141</width>
       <height>32</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Helvetica Neue</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
    <widget class="QPushButton" name="collidingSystemBut">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>110</y>
       <width>141</width>
       <height>32</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Helvetica Neue</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Colliding System</string>
     </property>
    </widget>
    <widget class="QPushButton" name="orbitingSystemBut">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>80</y>
       <width>141</width>
       <height>32</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Helvetica Neue</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Create Orbiting Sys</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>20</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Helvetica Neue</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>nBodySimulator</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>671</width>
      <height>251</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica Neue</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="Picture">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>0</y>
       <width>351</width>
       <height>251</height>
      </rect>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhNone</set>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../Documents/nBodyMainWindow.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>695</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
